viam_cartographer.go

```diff
@@ -123,7 +118,7 @@ func initSensorProcesses(cancelCtx context.Context, cartoSvc *CartographerServic
 	spConfig := sensorprocess.Config{
 		CartoFacade:              cartoSvc.cartofacade,
 		Lidar:                    cartoSvc.lidar,
-		IMU:                      cartoSvc.imu,
+		IMU:                      cartoSvc.movementSensor,
```

Comment:
Rename?

Reply:
Not yet, since this is the sensor process which will be handled in another ticket.